gdx/src/com/badlogic/gdx/graphics/g2d/GlyphLayout.java

```diff
@@ -140,7 +139,7 @@ else if (targetWidth <= fontData.spaceXadvance * 3) //
 				case '[':
 					// Possible color tag.
 					if (markupEnabled) {
-						int length = parseColorMarkup(str, start, end, colorPool);
+						int length = parseColorMarkup(str, start, end);
```

Comment:
`colorPool` is static and does not need to be in the parameters for this function.